x-pack/plugins/security_solution/public/hosts/pages/index.tsx

```diff
@@ -65,14 +65,29 @@ export const HostsContainer = React.memo(() => (
       }) => (
         <Redirect
           to={{
-            pathname: `${HOSTS_PATH}/${detailName}/${HostsTableType.authentications}`,
+            pathname: `${HOSTS_PATH}/name/${detailName}/${HostsTableType.authentications}`,
             search,
           }}
         />
       )}
     />
-
-    <Route
+    <Route // Compatibility redirect for the old user detail path.
```

Comment:
👌🏾 Not necessary now as this is a bug fix, but we should start adding telemetry to these so we can get a good idea of when we can remove logic like this